### ISSUE TITLE
S-Series Request data

### ISSUE THREAD

**Issue by frankol** [LucasTor/FoxESS-T-series#1]
hi,
im trying to get the real time data faster than the 300s delay the inverter is sending it.

Ive got the documentation about the protocol but im not smart enough to get it running. Maybe it will help you too, since the inverters are pretty similar (The T-Series integration is working for the s-series too)

The cloud server can request data from the inverter with function code 11 

Here is a sample string i sniffed

```
Header	function	     request string           length		                               checksum        header end
7F:7F 	11			C5:EC:C5:6C		00:06		01:04:27:74:00:18		6C:BF		F7:F7

Response:

Header	function	       requet string	        lengh		
7F:7F 	91			C5:EC:C5:6C		00:33		01:04:30:E0:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:2A:F7:7C:08:00:10:00:20:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00	7A:36		F7:F7
```
i dont understand the numbers after the holding register part. 
01 is the slave id, check
04 is the modbus holding register, check
but what 27:74 and 00:18 mean?
![image](https://github.com/LucasTor/FoxESS-T-series/assets/28876168/adc4640f-d376-476a-9faf-48b1a829c2fd)

i want all values but i dont know what i can send the inverter to get them

**Comment by LucasTor** [comment]
Hello @frankol, good to know the integration is working for you!
To be honest, I kind of skipped over the requesting data part of the documentation, I just decoded the payload it sends periodically, I read somewhere you can configure the interval time, If I'm not mistaken up to once every 5s, but I'll make sure to have a look at that part and possibly add this to the integration as well
Just for some context, are you using an Elfin EW11 as well?
Thanks for letting me know about that!

**Comment by frankol** [comment]
> Just for some context, are you using an Elfin EW11 as well?

No, i have connected a esp32 to the rs485 terminals of the included wifi adapter at the moment. But the s-series has a seperate rs485 port over a connector
